.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
I've remove the `.editorconfig`. I'm not sure why you wouldn't want one though. It's a pretty standard part of many open source projects. If you are unfamiliar with it the purpose is to allow anyone to work on the project without having to reconfigure their editor for tabs/spaces when switching between projects.